podcasts/Whats New/WhatsNewView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+struct WhatsNewView: View {
+    @EnvironmentObject var theme: Theme
+
+    let announcement: WhatsNewAnnouncement
+
+    var body: some View {
+        VStack(spacing: 10) {
+            ZStack(alignment: .top) {
+                Rectangle()
+                    .frame(height: 195)
+
+                HStack {
+                    Spacer()
+                    Button {
+                        NavigationManager.sharedManager.dismissPresentedViewController()
+                    } label: {
+                        ZStack {
+                            Image("close")
+                                .foregroundColor(.white)
+                        }
+                        .frame(width: 44, height: 44)
+                    }
+                }
+            }
+            Text(announcement.title)
+                .font(style: .title3, weight: .bold)
+                .padding(.horizontal)
+                .padding(.top)
+                .foregroundColor(theme.primaryText01)
+            Text(announcement.message)
+                .font(style: .subheadline)
+                .foregroundColor(theme.secondaryText02)
```

Comment:
If we use `secondaryText02` and `theme.primaryUi01` for the background then we'll run into issues where the contrast isn't great enough or the text is completely hidden for some themes:

Classic:

<img width="120" src="https://github.com/Automattic/pocket-casts-ios/assets/793774/f0e492df-ac33-4683-a825-ded0eb755bc8">

Indigo:

<img width="120" src="https://github.com/Automattic/pocket-casts-ios/assets/793774/2892ca8e-0563-4759-8e36-8771ff448ede">

The better value is `primaryText02`

```suggestion
                .foregroundStyle(theme.primaryText02)
```